### PR TITLE
[csp] Correct typo in documentation

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

**Comment by jugglinmike** [comment]
@hillbrad @andypaicu Mind taking a look?

**Comment by jugglinmike** [comment]
@foolip A change like this is probably safe to merge without approval from the test's maintainers. Would you mind taking a look?

**Review by foolip — APPROVED** [review]

**Pull request merged**
